src/Client/pages/WishList.fs

```diff
@@ -13,15 +13,16 @@ open Fable.Core.JsInterop
 open Fable.PowerPack
 open Fable.PowerPack.Fetch.Fetch_types
 
-type Model = { 
```

Comment:
I think original reason was it reduces diff if you add fields at beginnging or end ;-) 
Somehow it's used in many places